packages/office-ui-fabric-react/src/components/TeachingBubble/TeachingBubbleContent.tsx

```diff
@@ -114,7 +142,15 @@ export class TeachingBubbleContent extends BaseComponent<ITeachingBubbleProps, I
     }
 
     return (
-      <div className={css('ms-TeachingBubble-content', styles.root, isWide ? styles.wideCallout : null)}>
+      <div
+        ref={this.rootElement}
+        className={css('ms-TeachingBubble-content', styles.root, isWide ? styles.wideCallout : null)}
+        role={'dialog'}
+        tabIndex={-1}
+        aria-labelledby={ariaLabelledBy}
+        aria-describedby={ariaDescribedBy}
```

Comment:
these don't appear in the dom if they're undefined, correct?

Reply:
Correct, although they do appear in snapshot output